catalogue/webapp/components/VisuallySimilarImagesFromApi/VisuallySimilarImagesFromApi.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import { font, classNames } from '@weco/common/utils/classnames';
+import Image from '@weco/common/views/components/Image/Image';
+import { type Image as ImageType } from '@weco/common/model/catalogue';
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { getImage } from '../../services/catalogue/images';
+import Space from '@weco/common/views/components/styled/Space';
+
+type Props = {|
+  originalImage: ImageType,
+  onClickImage: (image: ImageType) => void,
+|};
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-end;
+
+  img {
+    margin-right: 10px;
+    margin-bottom: 10px;
+    height: auto;
+    max-height: 120px;
+    max-width: 190px;
+    width: auto;
+  }
+`;
+
+const VisuallySimilarImagesFromApi = ({
+  originalImage,
+  onClickImage,
+}: Props) => {
+  const [similarImages, setSimilarImages] = useState<ImageType[]>([]);
+  useEffect(() => {
+    const fetchVisuallySimilarImages = async () => {
+      const fullImage = await getImage({ id: originalImage.id });
+      setSimilarImages(fullImage.visuallySimilar);
+    };
+    fetchVisuallySimilarImages();
+  }, []);
+  return similarImages.length === 0 ? null : (
+    <Space v={{ size: 'xl', properties: ['margin-bottom'] }}>
+      <h3 className={font('wb', 5)}>Visually similar images</h3>
+      <Wrapper>
+        {similarImages.map(related => (
+          <a href="#" onClick={() => onClickImage(related)} key={related.id}>
+            <Image
+              contentUrl={related.locations[0] && related.locations[0].url}
+              defaultSize={250}
+              width={250}
+              alt=""
+              tasl={null}
+            />
+          </a>
+        ))}
+      </Wrapper>
+      <p
+        className={classNames({
+          [font('hnl', 6)]: true,
+          'no-margin': true,
+        })}
+      >
+        These images have similar shapes and structural features. We use machine
```

Comment:
I wonder if there is a robot icon we can use for messaging like this? @GarethOrmerod ?

Reply:
fwiw this is just copied and pasted from an existing component